### PR TITLE
Fixed UI for 640X480 and DragonBoard resolution

### PR DESCRIPTION
**Pull request by Sumahitha**

### CODE REVIEW

**Comment by anthony-ngu** [comment]
Have you tested this against the RPi2 and MBM resolutions?

File: SerialSample/CPP/Generated Files/XamlTypeInfo.Impl.g.cpp [hunk 1]
```diff
@@ -95,6 +95,41 @@ ::Windows::UI::Xaml::Markup::IXamlMember^ ::XamlTypeInfo::InfoProvider::XamlType
 }
 
 
+::Windows::UI::Xaml::Markup::IXamlType^ ::XamlTypeInfo::InfoProvider::XamlTypeInfoProvider::CheckOtherMetadataProvidersForName(::Platform::String^ typeName)
```
Comment (bfjelds) [inline SerialSample/CPP/Generated Files/XamlTypeInfo.Impl.g.cpp]: why is generated code checked in?

File: SerialSample/CPP/MainPage.xaml [hunk 2]
```diff
@@ -35,12 +35,12 @@
...
                         IsHitTestVisible="false" HorizontalAlignment="Center" VerticalAlignment="Center" Margin="10,10,0,0"/>
 
-        <StackPanel Grid.Row="1" Grid.Column="0" HorizontalAlignment="Center" VerticalAlignment="Center" Orientation="Horizontal" Margin="10,10,0,0">
+        <StackPanel Grid.Row="1" Grid.Column="0" HorizontalAlignment="Center" VerticalAlignment="Center" Orientation="Horizontal" Margin="70,10,0,0">
             <TextBlock Text="Select Device:" HorizontalAlignment="Left" VerticalAlignment="Top"/>
         </StackPanel>
 
-        <StackPanel Grid.Row="2" Grid.Column="0" HorizontalAlignment="Center" VerticalAlignment="Top" Orientation="Horizontal" Margin="10,0,0,0" Width="300" Height="80">
-            <ListBox x:Name="ConnectDevices" ItemsSource="{Binding Source={StaticResource DeviceListSource}}" Width="300" Height="80" Background="Gray">
+        <StackPanel Grid.Row="2" Grid.Column="0" HorizontalAlignment="Center" VerticalAlignment="Top" Orientation="Horizontal" Margin="70,0,0,0" Width="400" Height="80">
```
Comment (bfjelds) [inline SerialSample/CPP/MainPage.xaml:42]: why are we setting Grid.Column and a width?  shouldn't the column have the width?  also, does this app work if the width is less than 400?

Comment (Sumahitha) [inline SerialSample/CPP/MainPage.xaml:42]: this particular UI element is set to 400 to read the contents easily. We have verified the UI with various screens and it is consistent.

File: SerialSample/CPP/Generated Files/XamlTypeInfo.Impl.g.cpp (hunk 1 above)
Comment (Sumahitha) [inline SerialSample/CPP/Generated Files/XamlTypeInfo.Impl.g.cpp]: I tried to check-in the generated files that were already checked in. I did not want to remove any that were already there but I did not add any new ones

**Comment by Sumahitha** [comment]
These changes were tested on all 3 platforms and also reviewed by another team member

File: SerialSample/CPP/Generated Files/XamlTypeInfo.Impl.g.cpp (hunk 1 above)
Comment (bfjelds) [inline SerialSample/CPP/Generated Files/XamlTypeInfo.Impl.g.cpp]: the generated files shouldn't have been checked in to begin with ... please remove them if possible.

**Pull request merged**